### ISSUE TITLE
Metadata-override feature request

### ISSUE THREAD

**Issue by domenic** [tabatkins/bikeshed#1311]
(This is to fix https://github.com/whatwg/infra/issues/202.)

It seems like we need something that allows an in-document override or deletion of metadata, even if it was supplied by computed metadata.

As of the changes you helpfully made for us in #1161, right now any in-document metadata gets merged with the computed metadata. I think you alluded to the feature I'm now asking for when you said

> Let me know if I need to offer more functionality here and give you a choice between the two (like a "clear" option or something).

So yeah, something where we could get rid of the default WHATWG "!Tests" metadata from inside the Infra .bs file would be ideal.

**Comment by tabatkins** [comment]
Not quite, I'd actually need to add *another* metadata priority level, which comes after the computed-metadata defaults. Makes sense tho, so I'll get to it.

(Right now the order is group defaults, then document, then command line, then computed-metadata defaults.)

**Comment by annevk** [comment]
@tabatkins any chance this could be prioritized somehow?

**Comment by annevk** [comment]
This would still be great to have for Infra. I could maybe also give it a go if you can give some pointers.
